Dockerfile

```diff
@@ -23,4 +23,4 @@ ADD .travis.yml .
 ADD .rubocop.yml .
 ADD test /app/test
 
-CMD bundle exec bin/secrets
+CMD mkdir -p /secrets/bin && cp /app/bin/wait_for_it.sh /secrets/bin && bundle exec bin/secrets
```

Comment:
could be.  the thinking behind adding the script is that it could support more complex stuff in the future (also implements a timeout etc etc).  At some point we need to decide how much integration other people will need to do as we keep jumping back and forth over the line.  good quick topic for tomorrow (I'll be in the office).